ocp_utilities/infra.py

```diff
@@ -354,3 +357,69 @@ def create_icsp(icsp_name, repository_digest_mirrors):
     )
     icsp.deploy()
     return icsp
+
+
+def encode_secret_data(secret_data_dict):
```

Comment:
```suggestion
def dict_base64_encode(_dict):
```

Reply:
remove the word `secret`; not relevant in this function

Reply:
done